core/src/main/java/bio/terra/pearl/core/service/export/formatters/item/PropertyItemFormatter.java

```diff
@@ -55,6 +55,8 @@ public  Object getRawExportValue(T bean) {
         Object value = null;
         try {
             value = PropertyUtils.getNestedProperty(bean, propertyName);
+        } catch (NullPointerException | NestedNullException e) {
+            // this is expected if the property is null
```

Comment:
drive by fix for the "family.shortcode" null error that keeps popping up in logs (I really wish getNestedProperty would just return null if it didn't exist instead of throwing...)